Cargo.toml

```diff
@@ -28,9 +28,11 @@ openssl-sys = { version = "0.9.68", optional = true }
 [features]
 default = ['tor']
 # Mac OS needs to use the vendored version of openssl-sys as upstream `libtor` is not updated for to support mac os aarch (M series)
-build-macos-aarch = ["openssl-sys/vendored"]
+build-macos-aarch = ['openssl-sys/vendored', 'dep:libtor', 'dep:mitosis']
 # The following feature set is in response to the issue described at https://github.com/rust-lang/rust/issues/45599
 # Only used for running the integration tests
 integration-test = []
+# Mac OS needs to use the vendored version of openssl-sys for integrating testing as upstream `libtor` is not updated for to support mac os aarch (M series)
+integration-test-macos-aarch = ['openssl-sys/vendored']
 # Only used for running the tor
```

Comment:
Better framing
```suggestion
# Used for spawning Tor process and connecting to the Tor socks port`
```

We can later divide it into two separate features. Spawning and Connecting.

Not all users will need both. For some use cases, we may need only to connect tor socks port (mobile taker apps with Orbot). So mitosis can be avoided for those builds.

Reply:
Looks great, will update